addons/l10n_ro/data/template/account.account-ro.csv

```diff
@@ -240,121 +240,126 @@
+"pcg_47532","Investment donations - resumed in a period of at least one year","47532","income_other","False","Donații pentru investiții - reluate intr-o perioada mai mare de un an"
 "pcg_47541","Inventory pluses of the nature of fixed assets - resumed in a period of up to one year","47541","income_other","False","Plusuri de inventar de natura imobilizărilor - reluate intr-o perioada de pana la un an"
-"pcg_47542","Inventory pluses of the nature of fixed assets - resumed in a period of at least one year","47542","income_other","False","Plusuri de inventar de natura imobilizărilor - reluate intr-o perioada mal mare de un an"
+"pcg_47542","Inventory pluses of the nature of fixed assets - resumed in a period of at least one year","47542","income_other","False","Plusuri de inventar de natura imobilizărilor - reluate intr-o perioada mai mare de un an"
 "pcg_47581","Other amounts received as investment subsidies - resumed in a period of up to one year","47581","income_other","False","Alte sume primite cu caracter de subvenții pentru investiții - reluate intr-o perioada de pana la un an"
-"pcg_47582","Other amounts received as investment subsidies - resumed in a period of at least one year","47582","income_other","False","Alte sume primite cu caracter de subvenții pentru investiții - reluate intr-o perioada mal mare de un an"
+"pcg_47582","Other amounts received as investment subsidies - resumed in a period of at least one year","47582","income_other","False","Alte sume primite cu caracter de subvenții pentru investiții - reluate intr-o perioada mai mare de un an"
 "pcg_4781","Advance income related to assets received by transfer from customers - resumed in a period of up to one year","4781","income_other","False","Venituri în avans aferente activelor primite prin transfer de la clienți - reluate intr-o perioada de pana la un an"
-"pcg_4782","Advance income related to assets received by transfer from customers - resumed in a period of at least one year","4782","income_other","False","Venituri în avans aferente activelor primite prin transfer de la clienț - reluate intr-o perioada mal mare de un an"
+"pcg_4782","Advance income related to assets received by transfer from customers - resumed in a period of at least one year","4782","income_other","False","Venituri în avans aferente activelor primite prin transfer de la clienț - reluate intr-o perioada mai mare de un an"
 "pcg_481","Settlements between unit and subunits","481","liability_current","False","Decontări între unitate și subunități"
 "pcg_482","Settlements between subunits","482","liability_current","False","Decontări între subunități"
+"pcg_490","Allowances for depreciation of the receivables representing advances to suppliers","490","liability_current","False","Ajustări pentru deprecierea creanţelor reprezentând avansuri acordate furnizorilor"
```

Comment:
Should this one not be considered as an account.group instead ? It looks to be the "parent" class of the accounts below

Reply:
Yes it looks as a parent, but is present in SAF-T COA, so it must be present as an account. 

![image](https://github.com/user-attachments/assets/9937192f-5a80-4314-9cb9-12f89d9985b0)

https://static.anaf.ro/static/10/Anaf/Informatii_R/SAF_T_Ro_SchemaDefinitionCodes_v4_1_6_final_1712021.xlsx

Reply:
Not sure I get it, account group 49 is also present in the COA presented, but is an acount group because it has sub-accounts. This logic applies for all accounts, so it seems to me that 490 is considered an account group because there are accounts 4901, 4902, ...

Also, don't forget the commit messages on the 2 commits, so far you just have a message on the PR, but it won't be kept once merged